### PR TITLE
latest mailbox attributes feching task, & poll the inbox task files

### PR DESCRIPTION
**Pull request by tvbbchary**
please review the code , 
regarding the outlook project task1. ie ( reading the mail attributes of latest inbox mail, sent mail, deleted mail etc...
And the task 2 is , poll the latest inbox mail.

Thanks and regards
From chary

### CODE REVIEW

File: Chary/reading_mails_fixed_review_comments.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    inbox_folder = outlook.GetDefaultFolder(6)
+    folder_acces(inbox_folder)
+    sentbox_folder = outlook.GetDefaultFolder(5)
+    folder_acces(sentbox_folder)
+    drafts_folder = outlook.GetDefaultFolder(16)
+    folder_acces(drafts_folder)
+    deleted = outlook.GetDefaultFolder(3)
+    folder_acces(deleted)
+    outbox = outlook.GetDefaultFolder(4)
+    folder_acces(outbox)
```
Comment (mundlamurisivameghana) [inline Chary/reading_mails_fixed_review_comments.py]: what are the attributes you mentioned other day that your not getting.

**Comment by mundlamurisivameghana** [comment]
> please review the code ,
> regarding the outlook project task1. ie ( reading the mail attributes of latest inbox mail, sent mail, deleted mail etc...
> And the task 2 is , poll the latest inbox mail.
> 
> Thanks and regards
> From chary

Hi @tvbbchary what work are u doing today

File: Chary/reading_mails_fixed_review_comments.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+    def outbox_mail(self):
+        outbox_message = outbox.Items.GetLast()
+        mail_address = f" In outbox : {self.message_mail_address(outbox_message)}"
+        time = f"time stamp is: {self.message_time(outbox_message)}"
+        subject = f"subject is : {self.message_subject(outbox_message)}"
+        body = f" Body of the outbox mail is : {self.message_body(outbox_message)}"
+        outbox_latest_message = [mail_address, time, subject, body]
+        print(outbox_latest_message)
+        return outbox_latest_message
```
Comment (mundlamurisivameghana) [inline Chary/reading_mails_fixed_review_comments.py]: Hii @tvbbchary can optimize the code for above methods.

**Review by tvbbchary — COMMENTED** [review]
Good evening, 
Based the review comments, i fixed the issues...
please review the code.
Thanks and regards.
Chary

**Review by mundlamurisivameghana — APPROVED** [review]
Hi @tvbbchary you can merge these changes.

**Pull request merged**